doc/src/manual/profile.md

```diff
@@ -321,6 +327,15 @@ you want to analyze, then call [`Profile.clear_malloc_data()`](@ref) to reset al
  Finally, execute the desired commands and quit Julia to trigger the generation of the `.mem`
 files.
 
+### GC Logging
+
+While [`@time`](@ref) logs high-level stats about memory usage and garbage collection over the course
+of evaluating an expression, it can be useful to log each garbage collection event, to get an
+intuitive sense of how often the garbage collector is running, how long it's running each time,
+and how much garbage it collects each time. This can be enabled with
+[`Profile.enable_gc_logging(true)`](@ref), which causes Julia to log to stderr every time a
```

Comment:
```suggestion
[`Base.Experimental.enable_gc_logging(true)`](@ref), which causes Julia to log to stderr every time a
```

but maybe `Profile` is actually the better place

Reply:
oh whoops thanks

Reply:
I do agree with Valentin that `Base.Experimental` probably isn't actually the best home for this logging function.. Maybe either `Base.GC` or `Profile` would be preferable. @vchuravy do you have an opinion?

Reply:
On the other hand, I wonder if this should just go in `Base`, prefixed with `gc_` to be consistent with the other GC observability functions?:
```julia
julia> Base.gc_
gc_alignment    gc_alloc_count  gc_bytes        gc_live_bytes   gc_num          gc_time_ns      gc_total_bytes
```

This could be `Base.gc_enable_logging`?

@vchuravy / @vtjnash do you have a preference?

Reply:
`Base.GC.enable_logging`?

Reply:
Yeah, i like that too, but i did notice that currently the GC module is pretty bare, and all the observability stuff is Base.gc_*, as i mentioned above. Here's the GC module:
```julia
julia> Base.GC.
@preserve           GC_FULL             disable_finalizers  enable_finalizers   gc                  safepoint
GC_AUTO             GC_INCREMENTAL      enable              eval                include
```

That said, I think your suggestion is best, yeah. Is it okay to break that pattern? 😁

Reply:
Yeah it's okay to break the pattern. The rest should probably be in there (besides gc_alignment)

Reply:
Okay cool thanks. @vilterp can you move this one more time?